src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/zipfilestatus/ZipFileEvent.java

```diff
@@ -13,6 +15,7 @@
     public final String container;
 
     @JsonProperty("created_at")
+    @JsonSerialize(using = CustomTimestampSerialiser.class)
     public final Timestamp createdAt;
```

Comment:
I've removed `CustomTimestampSerialiser` in the latest pr https://github.com/hmcts/bulk-scan-processor/pull/466
Is it possible to serialize this date field using `@JsonFormat`?

Reply:
Looks like this class was not updated in #466 
Whatever gets in later will have to accommodate I guess.

Reply:
(as this also has to be an `Instant` not `Timestamp`